### PR TITLE
Add releases notes of SIG Monitoring

### PR DESCRIPTION
**Pull request by itrich**
Signed-off-by: Eduard Itrich <eduard@itrich.net>
Co-Authored-By: Ralf Heiringhoff <ralf@frosty-geek.net>
Co-Authored-By: Felix Kronlage-Dammers <fkr@hazardous.org>

### CODE REVIEW

File: Release-Notes/Release1.md [hunk 1]
```diff
@@ -17,19 +17,25 @@ the container layer with k8s Cluster API (now in version 0.4.x).
...
 ### openstack-health-monitor
 
 ### ...
 
-
 ## Logging
 
 ### Central logging
 
+OSISM now enables kolla-ansible centralized logging by default. The default rules need to be further refined to suit your needs. We plan to implement a more generic set of rules for R2.
```
Comment (itrich) [inline Release-Notes/Release1.md:38]: @berendt We weren't absolutely sure in the SIG Monitoring call today whether or not logging is now enabled by default in OSISM. Can you please confirm this?

**Comment by garloff** [comment]
I fear this will conflict with PR#11.

**Comment by garloff** [comment]
Actually, it does not, as PR #11 was in reality against a different document. The Release Notes should probably a summary of the monitoring doc.

**Comment by garloff** [comment]
Looks good, thanks!

**Review by garloff — COMMENTED** [review]
We can always do further refinement, but this is a very good step.

**Pull request merged**